README.md

```diff
@@ -335,3 +335,31 @@ val KernelBinary = cc.Binary{
 	Toolchain: kernelToolchain,
 }
 ```
+
+
+## Set up a local mirror
+
+DBT can read a configuration file located at:
+- `$DBT_CONFIG_DIR/config.yaml`, if `$DBT_CONFIG_DIR` is set.
+- `$XDG_CONFIG_DIR/dbt/config.yaml`, if `$XDG_CONFIG_DIR` is set.
+- `$HOME/.config/dbt/config.yaml`, if `$HOME` is set.
+- Otherwise no configuration is loaded.
+
+This configuration file allows the user to configure a DBT local mirror with the following line:
+
+```yaml
+mirror: "<PATH_TO_YOUR_LOCAL_MIRROR>"
+```
+
+Replace `<PATH_TO_YOUR_LOCAL_MIRROR>` by a full path in your system where your user has read and
+write permissions. Note that this path MUST exist for the mirror to be used by DBT.
+
+With a local mirror configured, DBT will reduce the amount of bandwidth required to sync dependencies.
+In particular, its behavior is different between archives and git repositories:
+- Compressed archives (`*.tar.gz`): they get downloaded first into the local mirror and then
+copied to your project's dependency folder. If they are already available in your local mirror, they
+are simply copied over to your dependency folder, so no network access is required.
```

Comment:
could hardlink them instead of copying to save disk space, but that might be for another time ;)

Reply:
I considered linking them but that could be problematic if they were changed manually by the user (not realizing it would affect other repos too). If we had a COW filesystem it would be nice though